root/themes/modal.ts

```diff
@@ -1,5 +1,5 @@
 // Modal
-import { h5, captionMedium } from './text';
+import { h5, captionMedium } from './text.theme';
```

Comment:
there is already `text.tsx` under `Typography`. Similarly other files in the folder can be renamed to add an extension `.theme` to make it more readable which files belong to which module.